src/main/kotlin/io/curity/identityserver/plugin/username/authentication/RequestModel.kt

```diff
@@ -19,6 +18,7 @@ class Post(request: Request)
 {
     @NotBlank(message = "validation.error.username.required")
     val username: String = request.getFormParameterValueOrError("username")
+    var register: MutableCollection<String>? = request.getFormParameterValues("register")
```

Comment:
Posting multiple values would be unexpected, right? I suggest using `getFormParameterValueOrError` and declaring the field as `String?`